experiments/generic/disk_fill/disk_fill_ansible_logic.yml

```diff
@@ -0,0 +1,78 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars:
+    c_util: "/chaoslib/litmus/disk_fill_by_litmus.yml"
+    c_container: "{{ lookup('env','TARGET_CONTAINER') }}"
+    c_experiment: "disk-fill"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    fill_percentage: "{{ lookup('env','FILL_PERCENT') }}"
+  
+  tasks:
+    - block:
+
+        ## GENERATE EXP RESULT NAME
+        - block:
+
+            - name: Construct chaos result name (experiment_name)
+              set_fact:
+                c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+
+          when: lookup('env','CHAOSENGINE')    
+
+        ## RECORD START-OF-EXPERIMENT IN LITMUSCHAOS RESULT CR
+        - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+          vars:
+            status: 'SOT'
+            namespace: "{{ a_ns }}"
+
+        ## PRE-CHAOS APPLICATION LIVENESS CHECK
+
+        - name: Verify that the AUT (Application Under Test) is running 
+          include_tasks: /utils/common/status_app_pod.yml
+          vars:
+            app_ns: "{{ a_ns }}" 
+            app_lkey: "{{ a_label.split('=')[0] }}"
+            app_lvalue: "{{ a_label.split('=')[1] }}"       
+            delay: 1
+            retries: 60
+
+        ## FAULT INJECTION 
+
+        - include_tasks: "{{ c_util }}"
+          vars:
+            namespace: "{{ a_ns }}"
+            label: "{{ a_label }}"
+            app_container: "{{ c_container }}"
+            action: "killapp"
+            fill_percent: "{{ fill_percentage }}"
+            chaos_duration: "{{ c_duration }}"
+
+        ## POST-CHAOS APPLICATION LIVENESS CHECK
+
+        - name: Verify AUT liveness post fault-injection
+          include_tasks: /utils/common/status_app_pod.yml
+          vars:
+            app_ns: "{{ a_ns }}" 
```

Comment:
case in point - these are carry overs from older litmus utils - whose vars need to be made consistent. We can take this up as a separate effort